yt_dlp/extractor/rtrfm.py

```diff
@@ -54,10 +49,11 @@ def _real_extract(self, url):
         display_id = self._match_id(url)
         webpage = self._download_webpage(url, display_id)
         show, date, title = self._search_regex(
-            [self._PLAY_SHOW, self._PLAY_SHOW_FROM],
+            r"\.playShow(?:From)?\('(?P<show>[^']+)', *'(?P<date>[0-9]{4}-[0-9]{2}-[0-9]{2})', *'(?P<title>[^']+)'(?:, \d+)?\)",
```

Comment:
this will be a bit more robust
```suggestion
            r"\.playShow(?:From)?\(\s*'(?P<show>[^']+)',\s*'(?P<date>[0-9]{4}-[0-9]{2}-[0-9]{2})',\s*'(?P<title>[^']+)'",
```
even better if you can also handle double quotes

Reply:
Done both.